invenio_rdm_records/services/github/release.py

```diff
@@ -32,6 +32,39 @@ def metadata(self):
         output = metadata.default_metadata
         output.update(metadata.extra_metadata)
         output.update(metadata.citation_metadata)
+
+        if not output.get("creators"):
+            # Get owner from Github API
+            owner = self.get_owner()
+            if owner:
+                output.update({"creators": [owner]})
+
+        # Default to "Unkwnown"
+        if not output.get("creators"):
+            output.update(
+                {
+                    "creators": [
+                        {
+                            "person_or_org": {
+                                "type": "personal",
+                                "family_name": "Unknown",
+                            },
+                        }
+                    ]
+                }
+            )
+        return output
+
+    def get_owner(self):
```

Comment:
Same as before, this should be contained inside the metadata class. This is about serializing the repo's owner information to RDM